### ISSUE TITLE
BUG: Selected Filters displayed in the pill containers on the Custom Case report are not able to be removed when using a keyboard only.

### ISSUE THREAD

**Issue by ttlenard** [ustaxcourt/ef-cms#8596]
**Describe the Bug**
While doing some accessibility testing, I noticed that on the Custom Case report, I can use my keyboard to tab through the filters and select items from the dropdown menus, but I am unable to remove selected filters using my keyboard once I add them.

**Business Impact/Reason for Severity**
low

**In which environment did you see this bug?**
Test

**Who were you logged in as?**
Any Court User

**What were you doing when you discovered this bug? (Using the application, demoing, smoke tests, testing other functionality, etc.)**
Testing other functionality

**To Reproduce**
1. Log in as any Court user
2. Click on Reports and select Custom Case report
3. Using only your keyboard, tab through the filters and made some selections from Case Status, Case Types, Assigned Juge, or Requested place of trial
4. Continue to tab through, and attempt to use your keyboard to remove a selection.
5. Tabbing on your keyboard doesn't ever allow the user to "select" any of the pill containers.

**Expected Behavior**
User should be able to navigate through the Custom Case report and add/remove filters using their keyboard only

**Actual Behavior**
A clear and concise description of what actually happened.
If a user is only using their keyboard on the custom case report, and they add in filters, they are unable to remove the filters that they chose using only their keyboard. They have to use their mouse to x out the pill container.

**Screenshots**
<img width="1032" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeEYwQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ec115aa0712e4a778235e03817134ef9c601c8a6/image.png" alt="image.png" />

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Cause of Bug, If Known**

**Process for Logging a Bug:**
* Complete the above information
* Add a severity tag (Critical, High Severity, Medium Severity or Low Severity). See below for priority definition. 

**Severity Definition:**
* Critical Defect
Blocks entire system's or module’s functionality
No workarounds available
Testing cannot proceed further without bug being fixed.

* High-severity Defect
Affects key functionality of an application
There's a workaround, but not obvious or easy
App behaves in a way that is strongly different from the one stated in the requirements

* Medium-severity Defect
A minor function does not behave in a way stated in the requirements.
Workaround is available and easy

* Low-severity Defect
Mostly related to an application’s UI
Doesn't need a workaround, because it doesn't impact functionality

## Definition of Ready for Bugs(Created 10-4-21)
Definition used: A failure or flaw in the system which produces an incorrect or undesired result that deviates from the expected result or behavior. (Note: Expected results are use cases that have been documented in past user stories as acceptance criteria and test cases, and do not include strange behavior unrelated to use cases.)

The following criteria must be met in order for the development team to begin work on the bug.

The bug must:
 - Be focused on solving a user problem
 - Contain data for all fields in the bug template, so the team can pick it up and begin working immediately

Process: If the unexpected results are new use cases that have been identified, but not yet built, new acceptance criteria and test cases should be captured in a new user story and prioritized by the product owner. 

If the Court is not able to reproduce the bug, add the “Unable to reproduce” tag. This will provide visibility into the type of support that may be needed by the Court. In the event that the Court cannot reproduce the bug, the Court will work with Flexion to communicate what type of troubleshooting help may be needed.

## Definition of Done (Updated 4-14-21)
**Product Owner**
 - [ ]  Bug fix has been validated in the Court's test environment

**Engineering**
 - [ ] Automated test scripts have been written
 - [ ] Field level and page level validation errors (front-end and server-side) integrated and functioning
 - [ ] Verify that language for docket record for internal users and external users is identical
 - [ ] New screens have been added to pa11y scripts
 - [ ] All new functionality verified to work with keyboard and macOS voiceover https://www.apple.com/voiceover/info/guide/_1124.html 
 - [ ] READMEs, other appropriate docs and swagger/APIs fully updated
 - [ ] UI should be touch optimized and responsive for external only (functions on supported mobile devices and optimized for screen sizes as required)
 - [ ] Interactors should validate entities before calling persistence methods
 - [ ] Code refactored for clarity and to remove any known technical debt
 - [ ] Deployed to the Court's `test` environment if prod-like data is required. Otherwise, deployed to any `experimental` environment for review.
